### ISSUE TITLE
 pip install guildai on macos x fails

### ISSUE THREAD

**Issue by mdiephuis** [guildai/guildai#133]
Runnning
    
    pip install guildai
    
ERROR: Could not find a version that satisfies the requirement guildai (from versions: none)
ERROR: No matching distribution found for guildai
    
    pip --verbose search guildai`
    
Starting new HTTPS connection (1): pypi.org:443
https://pypi.org:443 "POST /pypi HTTP/1.1" 200 247
guildai (0.6.6)  - Toolkit to automate and optimize machine learning model development

    
    pip --version
    
pip 20.0.2 from /Users/Maurits/Development/virtual_env/ML/lib/python3.7/site-packages/pip (python 3.7)

    
    uname -a
    
Darwin Miauw.local 17.7.0 Darwin Kernel Version 17.7.0: Thu Jun 21 22:53:14 PDT 2018; root:xnu-4570.71.2~1/RELEASE_X86_64 x86_64

    
    tf.__version__

'2.1.0'

**Comment by gar1t** [comment]
Could you try installing the pre release version?

```
pip install guildai --pre
```

If that doesn't work could you attach the out from `pip debug --verbose guildai`? It might be a lot of output so if you want to create a gist or attach it as a file, that's fine.

Thanks for the report!

**Comment by mdiephuis** [comment]
pip debug --verbose guildai
    
ERROR: Could not find a version that satisfies the requirement guildai (from versions: none)
ERROR: No matching distribution found for guildai

    
    pip debug --verbose guildai
    

[gist with output](https://gist.github.com/mdiephuis/b209a437e1475cdea4ff0f7527a46c2e)

Many thanks!

**Comment by gar1t** [comment]
We don't have a release available for macOS 10.13. We do for 10.14 and 10.15.

I'll look into whether it's possible to create a release that's less picky about macOS version. Guild uses some natively compiled libraries so is specific to a platform architecture and system library version.

You could try installing the wheel for 10.14 but my guess is that some functionally will be broken. If want to try, download the wheel for your version of Python from here:

https://pypi.org/project/guildai/0.7.0.rc8#files

And install it using pip.

Then run `guild check --verbose` to check the install.

**Comment by mdiephuis** [comment]
Many thanks, I think will upgrade my os x.

**Comment by MislavSag** [comment]
I have reinstall anaconda and try to install guildai again with `pip install guildai` and `pip install guildai --user` but I got the error:
```
ERROR: Could not find a version that satisfies the requirement guildai (from versions: none)
ERROR: No matching distribution found for guildai
```
Result of `pip debug --verbose guildai`:
```
ERROR: Could not find a version that satisfies the requirement guildai (from versions: none)
ERROR: No matching distribution found for guildai
(base) PS C:\Users\Mislav> pip debug --verbose guildai
WARNING: This command is only meant for debugging. Do not use this with automation for parsing and getting these details, since the output and options of this command may change without notice.
pip version: pip 20.2.2 from c:\programdata\anaconda3\lib\site-packages\pip (python 3.8)
sys.version: 3.8.3 (default, Jul  2 2020, 17:30:36) [MSC v.1916 64 bit (AMD64)]
sys.executable: c:\programdata\anaconda3\python.exe
sys.getdefaultencoding: utf-8
sys.getfilesystemencoding: utf-8
locale.getpreferredencoding: cp1252
sys.platform: win32
sys.implementation:
  name: cpython
'cert' config value: Not specified
REQUESTS_CA_BUNDLE: None
CURL_CA_BUNDLE: None
pip._vendor.certifi.where(): c:\programdata\anaconda3\lib\site-packages\pip\_vendor\certifi\cacert.pem
pip._vendor.DEBUNDLED: False
vendored library versions:
  appdirs==1.4.4
  CacheControl==0.12.6
  colorama==0.4.3
  contextlib2==0.6.0.post1 (Unable to locate actual module version, using vendor.txt specified version)
  distlib==0.3.1
  distro==1.5.0 (Unable to locate actual module version, using vendor.txt specified version)
  html5lib==1.1
  ipaddress==1.0.23
  msgpack==1.0.0 (Unable to locate actual module version, using vendor.txt specified version)
  packaging==20.4
  pep517==0.8.2
  progress==1.5
  pyparsing==2.4.7
  requests==2.24.0
  certifi==2020.06.20
  chardet==3.0.4
  idna==2.10
  urllib3==1.25.9
  resolvelib==0.4.0
  retrying==1.3.3 (Unable to locate actual module version, using vendor.txt specified version)
  setuptools==44.0.0 (Unable to locate actual module version, using vendor.txt specified version)
  six==1.15.0
  toml==0.10.1
  webencodings==0.5.1 (Unable to locate actual module version, using vendor.txt specified version)
Compatible tags: 30
  cp38-cp38-win_amd64
  cp38-abi3-win_amd64
  cp38-none-win_amd64
  cp37-abi3-win_amd64
  cp36-abi3-win_amd64
  cp35-abi3-win_amd64
  cp34-abi3-win_amd64
  cp33-abi3-win_amd64
  cp32-abi3-win_amd64
  py38-none-win_amd64
  py3-none-win_amd64
  py37-none-win_amd64
  py36-none-win_amd64
  py35-none-win_amd64
  py34-none-win_amd64
  py33-none-win_amd64
  py32-none-win_amd64
  py31-none-win_amd64
  py30-none-win_amd64
  cp38-none-any
  py38-none-any
  py3-none-any
  py37-none-any
  py36-none-any
  py35-none-any
  py34-none-any
  py33-none-any
  py32-none-any
  py31-none-any
  py30-none-any
```
Additional info:
```
(base) PS C:\Users\Mislav\Documents\GitHub\trademl> pip --verbose search guildai
Getting credentials from keyring for pypi.org
Starting new HTTPS connection (1): pypi.org:443
https://pypi.org:443 "POST /pypi HTTP/1.1" 200 251
guildai (0.7.0.post1)  - Toolkit to automate and optimize machine learning model development
```
and
```
(base) PS C:\Users\Mislav\Documents\GitHub\trademl> uname -a
MSYS_NT-10.0-18363 DESKTOP-KBV1GE9 3.0.7-338.x86_64 2019-09-24 23:47 UTC x86_64 Msys
```

**Comment by gar1t** [comment]
@MislavSag Guild is not available for Windows 3.8. I'll see what I can do about that - from what I can see there's no good reason for this (when the decision was made TensorFlow didn't support 3.8 on Windows, which may have contributed to the decision).

In the meantime, you should be able to run on Windows with Python 3.6 or 3.7.

Stay tuned for 3.8 support.
